### PR TITLE
Improve visual consistence of the app

### PR DESCRIPTION
**Pull request by MathieuxC**
This PR change the app so to improve visual consistence, especialy regarding colors and TopAppBar. The colors to use are in Color.kt and can be accessed using `MaterialTheme.colorScheme.<color you want>`

### CODE REVIEW

**Review by AKallala — CHANGES_REQUESTED** [review]
Great job going through all these files and doing the modifications. I just made some remarks and suggestions regarding the colors. Also I don't understand why do you didn't use the same top bar for all screens and why did you add the bottom bar to some secondary screens but not to others: for example you added the bottom bar to the pdf converter screen and the todo list but not to the pomodoro or the calculator screens

File: app/src/main/java/com/github/se/eduverse/ui/folder/DisplayFolder.kt [hunk 1]
```diff
@@ -126,7 +128,8 @@ fun FolderScreen(
       floatingActionButton = {
         FloatingActionButton(
             onClick = { navigationActions.navigateTo(Screen.CREATE_FILE) },
-            modifier = Modifier.testTag("createFile")) {
+            modifier = Modifier.testTag("createFile"),
+            backgroundColor = MaterialTheme.colorScheme.tertiaryContainer) {
```
[the remark this answers is not in the document]
Comment (MathieuxC) [reply app/src/main/java/com/github/se/eduverse/ui/folder/DisplayFolder.kt:132]: Yes I noticed but didn't have the time to solve it yestedray, it is done now

File: app/src/main/java/com/github/se/eduverse/ui/folder/DisplayFolder.kt [hunk 2]
```diff
@@ -87,8 +89,8 @@ fun FolderScreen(
             modifier = Modifier.testTag("topAppBar"),
```
[the remark this answers is not in the document]
Comment (MathieuxC) [reply app/src/main/java/com/github/se/eduverse/ui/folder/DisplayFolder.kt:89]: Yes I didn't use the same as for the rest of the app cause I thought it looked weird in white, but using this color for the whole app is a good idea

File: app/src/main/java/com/github/se/eduverse/ui/converter/PdfConverter.kt [hunk 3]
```diff
@@ -245,7 +250,10 @@ fun OptionCard(
               .clickable(onClick = onClick, enabled = optionEnabled)
               .testTag(testTag),
       elevation = CardDefaults.cardElevation(8.dp),
-      colors = CardDefaults.cardColors(containerColor = Color(0xFFC2F5F0))) {
+      colors =
```
[the remark this answers is not in the document]
Comment (MathieuxC) [reply app/src/main/java/com/github/se/eduverse/ui/converter/PdfConverter.kt:253]: Yes, I don't know why the elevation turned this way, it is good now

File: app/src/main/java/com/github/se/eduverse/ui/search/UserProfileScreen.kt [hunk 4]
```diff
@@ -176,13 +181,15 @@ fun UserProfileScreen(
...
+                    icon = { Icon(Icons.Default.Article, contentDescription = null) },
+                    selectedContentColor = MaterialTheme.colorScheme.primary)
                 Tab(
                     selected = selectedTab == 1,
                     onClick = { selectedTab = 1 },
                     modifier = Modifier.testTag("favorites_tab"),
                     text = { Text("Favorites") },
-                    icon = { Icon(Icons.Default.Favorite, contentDescription = null) })
+                    icon = { Icon(Icons.Default.Favorite, contentDescription = null) },
+                    selectedContentColor = MaterialTheme.colorScheme.primary)
```
[the remark this answers is not in the document]
Comment (MathieuxC) [reply app/src/main/java/com/github/se/eduverse/ui/search/UserProfileScreen.kt]: Done

**Review by AKallala — CHANGES_REQUESTED** [review]
Well done, it looks much better this way. Thank you for your work. There's only a small thing left to change, it's the color of the circular progression of the pomodoro timer I think it's better if we set it to some blue color, that way it's in better harmony with the rest of the app's theme

**Review by AKallala — APPROVED** [review]
It's all good now, well done !

**Pull request merged**